content/chat-quick-start/js.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Skygear Chat Quick Start
+---
+
+The following quick start guide assumes that you are working on a new Node.js project with Skygear template project using Skygear cloud services.
+
+The template project will configure the Skygear and the Skygear SDK for you so that you can start working on your app directly.
+
+If you want to use Skygear Chat JS SDK with other JS projects (e.g. Ionic, React Native, etc), please check out the [Skygear JS guide](https://docs.skygear.io/guides/get-started/js/).
+
+## Prerequisite
+
+#### 1. Create an account on [the Skygear portal](https://portal.skygear.io/signup)
+
+Upon signing up, you can find the API keys and the server endpoints in _info_ page. You need them to configure your Skygear app.
+
+#### 2. Install Node.js
+
+We need Node.js to run the template project. If you have never installed Node.js, install it as follow:
+
+```bash
+# for MacOS
+brew install node
+
+# for Debian/Ubuntu
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+Visit [Node.js setup guide](https://nodejs.org/en/download/) for instructions if you are using a different operating system.
+
+## Setting up the template project
+
+Let's install and run the [Yeoman Skygear scaffolding generator](https://github.com/SkygearIO/generator-skygear) to set up the template project. React, Babel ES6 and Webpack will be bundle-installed.
+
+```bash
+# install yeoman and skygear generator globally
+npm install -g yo
+npm install -g generator-skygear
+
+# create your project folder
+mkdir new-skygear-project
+cd new-skygear-project
+
+# generate your project and follow on screen instruction
+yo skygear
+
+# answer the following questions to configure Skygear
+# You can find the API key and the endpoint in your Skygear portal
+? What is your skygear endpoint? https://<your-app-name>.skygeario.com/
```

Comment:
can refer to `lib/skygear-getstarted/src/pages/js-new.md` for formatting the user input here